src/ripple/app/main/Main.cpp

```diff
@@ -190,7 +190,7 @@ int run (int argc, char** argv)
 
     using namespace std;
 
-    setCallingThreadName ("main");
+    beast::setCurrentThreadName ("main");
```

Comment:
This would previously have set the thread's name to `rippled: main` but will now set it, only, to `main`.